.meta/sinks/papertrail.toml

```diff
@@ -0,0 +1,34 @@
+[sinks.papertrail]
```

Comment:
Should we consider renaming this to `papertrail_logs`?

Reply:
As far as I know they don't accept metrics so I would lean no?